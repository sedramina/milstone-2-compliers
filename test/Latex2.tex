\documentclass{article}
\usepackage[utf8]{inputenc}



\begin{document}
\title{Second Template}
\date{March 2016}
\maketitle


\section{Equation}
In this template you have the equation environment in latex. It is an alternative for the math mode but with a larger scope. Moreover, in the equation mode we have the label command that adds a label for each equation that you can later use to reference an equation. \\
\\
\begin{equation}
\label{eq1}
2x-5y=8
\end{equation}
\\
\begin{equation}
\label{eq2}
f(x)=x^2
\end{equation}


%This line is commented and it should not be shown in your output

\section{Errors}
In latex you cant have a backslash within a text, otherwise, it will cause an error as undefined control sequence. In this milestone you have to include the line number that caused the error in the error message. \errorcoming


\end{document}