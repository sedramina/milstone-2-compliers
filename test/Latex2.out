Second Template
March 2016
Equation
In this template you have the equation environment in latex.  It is an alternativefor the math mode but with a larger scope.  Moreover, in the equation mode we have the label command that adds a label for each equation that you can later use to reference an equation.

2x-5y= 8

f(x) =x^2
Errors
In  latex  you  cant  have  a  backslash  within  a  text,  otherwise,  it  will  cause  an undefined control sequence error.  In this milestone you have to include the line number that caused the error in the error message.
Parse Error
